### DISCUSSION TITLE
3p Wärmepumpe mit schlisser Kontakt 8 grad höher fahren mit SHM 2.0 ?

### DISCUSSION THREAD

**Discussion by DrMabuse100** [camueller/SmartApplianceEnabler#310] [Allgemein]
Hallo ich möchte meine 3-Phasige Wpum 8 grad höher fahren wenn PV überschuß anliegt.
Es gibt einen 3p zähler für die WP. der hat einen S0 ausgang der aber leider für eine andere anwendung benötigt wird.
Ist es möglich eine S0 aufzuspiltten, dann hätte man ja schon die leistung für den SAE zu verfügung oder ?
Und wie generie ich den kontakt für die WP. ?

Freundlicher Gruß

**Comment by camueller** [comment]
Innerhalb des SAE kann die [Zuordnung eines Zählers zu verschiedenen Geräten erfolgen](https://github.com/camueller/SmartApplianceEnabler/blob/2.0/doc/MasterSlaveMeter_DE.md).
Im Zusammenspiel mit anderen Anwendungen ist das nicht möglich.

**Comment by DrMabuse100** [comment]
Ok Vielen dank also heist das wenn ich den S0 bus noch für andere anwendungen brauche muss ich einen zeiten 3p zähler einschleifen ?
Birnge ich dann den S0 bus werte für 3p über deb SAE richtig in das Sunnyportal ?

Danke Freundlicher Gruß
